App.tsx

```diff
@@ -248,9 +256,20 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                             IconClass={Notifications}
                             iconColor={PXBColors.red[500]}
                             title={'1 Alarm'}
+                            fontColor={theme.dark ? PXBColors.red[200] : PXBColors.red[500]}
+                        />
+                        <InfoListItem
+                            dense
+                            IconClass={Info}
+                            iconColor={theme.dark ? PXBColors.blue[200] : PXBColors.blue[500]}
+                            title={'1 Event'}
+                        />
+                        <InfoListItem
+                            dense
+                            IconClass={Clouds}
+                            title={'Online'}
+                            iconColor={theme.dark ? PXBColors.black[200] : theme.colors.text}
```

Comment:
These should be fixed/updated in the component library logic. Let's add a @TODO so we remember to come back and pull these out.